src/index.css

```diff
@@ -2,5 +2,57 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: 'Open Sans', sans-serif;
+  font-family: "Open Sans", sans-serif;
+}
+
+/* body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+} */
+
+.d-flex {
+  display: flex;
+}
+
+.navbar {
+  background-color: #444;
+  color: #fff;
+  padding: 0 15%;
+  font-weight: 600;
+}
```

Comment:
-Please make the `.navbar` `display: fixed` so that it will stick to the top of the page.

Reply:
Thanks for your valuable feedback @addisalemseifu  as you said make the nav as display fixed I thank it is not a part of the requirement so please check the code once more if there is something to change let me know. thanks